Automattic/sensei
# Use block.json for metadata shared between JS and PHP

This makes it easier to keep block settings - mostly the attributes - in sync between JS and PHP block registration. This is the same pattern WP core / Gutenberg uses. 

This only includes the Outline parent block, will have another one with the Lesson and Module blocks once work in those is done. 

### Changes proposed in this Pull Request

* Move the Course outline parent block files into a folder
* Move shared block settings into a `block.json` file
* Move block registration out from the block itself — this could help with testing
* Fix placeholder icon for Course outline parent block

### Testing instructions

* Run `npm install` to update webpack dependencies 
* Make sure Outline block in editor still works
* Make sure Outline in frontend still works
